docs/eog/case-studies-cw20-risk-management.rst

```diff
@@ -0,0 +1,265 @@
+- Do not know where help is needed and what their duties are at the event
+- Feel unappreciated or excluded
+- Some members being unable to help during the event due to insufficient equipment or other implications arising due to lockdown
+- Members having connection or technical issues preventing them from carrying out their duties
+
+The strategy we took to mitigate these risks included:
+
+- Define different roles and let people choose what they want to do, communicate clearly what is expected from each role
+- Have backups for every event role
+- Devise a duties roster for the event where everyone is assigned tasks and communicate it ahead of the event
+- Thank our Organising Committee members during the event, make sure they are made known to the audience and feel appreciated
+
+Risks that we hadn't considered but that could have been an issue included sickness or being unable to present due to caring or other responsibilities imposed based on lockdown restrictions, but our mitigation strategy still applies to these scenarios.
+
+
+Volunteers/helpers at the event
+^^^^^^^^^^^^
+
+The risks identified regarding volunteers/helpers at the event included:
+
+- Not having enough volunteers/helpers
+- Volunteers/helpers having connection, technical or other issues preventing them from carrying out their duties
+- Volunteers/helpers feeling unappreciated or excluded
+
+The strategy we took to mitigate these risks included:
+
+- Reach out to our community and recruit more people with experience organising online events
+- Recruit enough volunteers/helpers in order to have backups for every event role
+- Communicate expectations clearly and make a duties roster for the event to make sure everyone knows what they are doing
+- Make sure they are made known to the audience, thanked during the event and feel appreciated
+
+
+Budget 
+--------------------
+
+As part of the initial discussion with the main stakeholder, we considered the following with regard to our finances: 
+
+- What are our committed expenses, lost income (sponsorship, registration), expenses incurred for other things such as refunding Fellows 
+
+   * What proportion of people are we supporting to attend? 
+   * How much will we have to reimburse for travel?
+   
+- Are there any additional costs to modify the event?
+
+As part of our contingency assessment, we calculated the financial risk and projected balance for the three scenarios: 
+
+1. Carry on in person
+
+   - No change in sponsorship
+   - Might lose some registrations
+   - No change in what we pay for staff/fellows hotels, although some may now not attend
+
+2. Move CW20 online
+
+   - We would have to ask sponsors
+   - We might lose some registrations
+   - We would have to change what we charge from £250 for a full in-person ticket to £50 for an online pass to attend (to go towards invested staff time and prizes)
+   - There would be no additional costs as we already had the video conferencing infrastructure in place
```

Comment:
Is it worth mentioning here - the potential loss of money from cancelling the venue, accommodation and reimbursing peoples travel that we had agreed?